### DISCUSSION TITLE
Welcome to sce-greenbutton Discussions!

### DISCUSSION THREAD

**Discussion by gauthig** [gauthig/greenbutton-influxdb#10] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by DjAnu** [comment]
I figured this is probably the right place to post this instead of reporting it in Issues : 


Hi This is not an issue, Just trying to reach out to you for help. I am not a programmer but I found your while looking for a way to read from SCE Green button data. Your code provide excellent information and might be useful for a lot of user. But not being a programmer I couldn't understand how to use it. I have a Docker installed on my synology and am also using Grafana to get data from Teslamate. Can you help me with using your tool. I can then write detailed instructions for general community. Thanks.
